.github/workflows/integration-tests.yml

```diff
@@ -153,7 +156,7 @@ jobs:
             nodes: 1
             os: ubuntu-latest
           - name: ocr2vrf
-            nodes: 2
+            nodes: 1
```

Comment:
ocr2vrf still has 2 tests so I don't think we want to reduce the parallel test count here.